apps/admin/backend/src/server.ts

```diff
@@ -70,24 +51,38 @@ export async function start({
         'workspace path could not be determined; pass a workspace or run with ADMIN_WORKSPACE'
       );
     }
-    /* istanbul ignore next */
     resolvedWorkspace = createWorkspace(workspacePath);
   }
 
-  // clear any cached data
+  // Clear any cached data
   resolvedWorkspace.clearUploads();
 
-  const usb: Usb = { getUsbDrives };
-
+  let resolvedApp = app;
   /* istanbul ignore next */
-  const resolvedApp =
-    app ??
-    buildApp({
+  if (!resolvedApp) {
+    const auth = new DippedSmartCardAuth({
+      card:
+        isFeatureFlagEnabled(BooleanEnvironmentVariableName.USE_MOCK_CARDS) ||
+        isIntegrationTest()
+          ? new MockFileCard()
```

Comment:
I see that this isn't new, but I find myself wondering: what happens if someone runs `rm -rf /tmp/*`? It appears that when anything tries to read from the mock card that it'll just throw an exception.

Reply:
Yup you're right - the `getAuthStatus` call will fail and the UI will crash if someone runs `rm -rf /tmp/*` while using mock cards. Let me open a ticket to fix this. Should be as simple as creating the file if it doesn't exist!